R/run.R

```diff
@@ -186,14 +186,29 @@ run_bm <- function(bm, ..., n_iter = 1, profiling = FALSE, global_params = list(
   defaults <- lapply(get_default_args(bm$setup), head, 1)
   defaults$cpu_count <- parallel::detectCores()
   params <- modifyList(defaults, list(...))
-  params <- modifyList(params, global_params)
-  params$packages <- package_info()[, c("package", "loadedversion", "date", "source")]
-  names(params$packages)[2] <- "version"
-
-  structure(list(
-    result = do.call(rbind, results),
-    params = params
-  ), class = "arrowbench_result")
+  all_params <- modifyList(params, global_params)
+  all_params$packages <- package_info()[, c("package", "loadedversion", "date", "source")]
+  names(all_params$packages)[2] <- "version"
```

Comment:
we can assume that "loadedversion" will always be at this position right?

Reply:
Should be ok because of the previous line. I don't love the hardcoded index, but this is only part of the diff because I changed the variable name. Open to a cleaner alternative